{"error":{"message":"Operation `users.findOne()` buffering timed out after 10000ms","name":"MongooseError","stack":"MongooseError: Operation `users.findOne()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (E:\\planning-pocker\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\collection.js:185:23)\n    at listOnTimeout (node:internal/timers:594:17)\n    at process.processTimers (node:internal/timers:529:7)"},"level":"error","message":"Erreur lors de la création des utilisateurs de test","timestamp":"2025-06-11 22:41:51"}
{"error":{},"level":"error","message":"Erreur lors de la connexion","timestamp":"2025-06-11 22:41:58"}
{"error":{"message":"Command find requires authentication","name":"MongoServerError","stack":"MongoServerError: Command find requires authentication\n    at Connection.onMessage (E:\\planning-pocker\\node_modules\\mongodb\\lib\\cmap\\connection.js:202:26)\n    at MessageStream.<anonymous> (E:\\planning-pocker\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:518:28)\n    at processIncomingData (E:\\planning-pocker\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (E:\\planning-pocker\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:572:12)\n    at _write (node:internal/streams/writable:501:10)\n    at Writable.write (node:internal/streams/writable:510:10)\n    at Socket.ondata (node:internal/streams/readable:1009:22)\n    at Socket.emit (node:events:518:28)"},"level":"error","message":"Erreur lors de la création des utilisateurs de test","timestamp":"2025-06-11 22:49:23"}
{"error":{"code":13,"codeName":"Unauthorized","ok":0},"level":"error","message":"Erreur lors de la connexion","timestamp":"2025-06-11 22:49:39"}
